.github/workflows/build_test_invoke.yml

```diff
@@ -122,6 +122,13 @@ jobs:
           - version: '11'
             type: 'Invoke (AL2)'
             file: 'tests/integration/build_invoke/java/test_java_11_al2.py'
+#          TODO: Enable when build image will be public
```

Comment:
do we need new PR for resolve this TODO ?

Reply:
I think running unit tests should be fine, but for build & invoke, we need the new build image. I am testing the unit tests by un-commenting it right now.